app/src/main/res/layout/content_no_tickets.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="@dimen/layout_margin_extra_large"
+    android:orientation="vertical">
+
+    <RelativeLayout
+        android:layout_width="@dimen/item_image_view_large"
+        android:layout_height="@dimen/item_image_view_large"
+        android:layout_gravity="center_horizontal"
+        android:layout_marginTop="@dimen/layout_margin_large">
+
+        <ImageView
+            android:layout_width="@dimen/item_image_view_large"
+            android:layout_height="@dimen/item_image_view_large"
+            android:layout_centerInParent="true"
+            android:scaleType="centerCrop"
+            app:srcCompat="@drawable/round_shape" />
+
+        <android.support.v7.widget.AppCompatImageView
```

Comment:
ImageView

Reply:
> ImageView

But it is showing error with ImageView `Unexpected namespace prefix "app" found for tag ImageView`

Reply:
But it is working fine. Updated